pkg/exporter/process.go

```diff
@@ -98,6 +101,60 @@ func InitExportingProcess(collectorAddr net.Addr, obsID uint32, tempRefTimeout u
 	return expProc, nil
 }
 
+func InitExportingProcessWithTLS(collectorAddr net.Addr, obsID uint32, cert []byte) (*ExportingProcess, error) {
```

Comment:
Here as well taking the input for encryption may be better. We can save some duplication of code and make this library function simpler.